### ISSUE TITLE
Angular Universal changelog file not updated

### ISSUE THREAD

**Issue by Twois** [angular/universal#1620]
- [ ] aspnetcore-engine
- [ ] builders
- [ ] common
- [ ] express-engine
- [ ] hapi-engine
- [ ] module-map-ngfactory-loader
- [X] Documentation

### Is this a regression?
Hard to follow what the available features, fixes.

### Description
The changelog not updated for a while, it was good way to follow the changes.

**Comment by alan-agius4** [comment]
Thanks for pointing this out. 

In the mean time you can view the list of changes in the releases page.

https://github.com/angular/universal/releases

**Comment by hiepxanh** [comment]
```
commit 38e5ce2
Author: Renovate Bot bot@renovateapp.com
Date: Fri Mar 27 05:13:55 2020 +0000

build: update codelyzer to version 5.2.2
```
I think this information make other people hard to follow, it not necessary in changelogs

**Comment by alan-agius4** [comment]
Done.
